docs/ROADMAP.md

```diff
@@ -1,3 +1,38 @@
+## 2020 Goals and Focus
+
+### Goal#1 - Ensure our framework and docs are ready for LB3 users to migrate to LB4
+
+With LoopBack 3 going end-of-life at the end of 2020, we are continuing to focus
+on the following areas:
+
+- provide a migration guide to help LB3 users to migrate their applications to
+  LB4
+- offer some level of tooling support to make the migration is easier
+- fill in feature parity gaps by either implementing the features in LB4 or
+  recommending a solution using existing libraries
+- add APIC/LB4 integration
```

Comment:
Let's replace acronyms with full names here, I am concerned this line can be too cryptic for people outside of our team.

```suggestion
- Integrate LoopBack 4 into IBM API Connect 
```